src/assets/CloudinaryTransformable.ts

```diff
@@ -80,12 +81,22 @@ class CloudinaryTransformable extends CloudinaryFile {
   }
 
   /**
-   * @desc A proxy to {@link SDK.Transformation| Transformation} - Calls the same method contained in this.transformation
-   * @param {Actions.Delivery} quality
+   * @desc An alias to Action Delivery.quality
+   * @param {string} quality
+   * @return {this}
+   */
+  quality(quality: string): this {
+    this.addAction(new DeliveryFormat('q', quality));
+    return this;
+  }
+
+  /**
+   * @desc An alias to Action Delivery.format
+   * @param {string} format
    * @return {this}
    */
-  quality(quality: DeliveryQualityAction): this {
-    this.transformation.quality(quality);
+  format(format: string): this {
```

Comment:
In general this should accept anything of that type Format.{thing} -

which is number, string, auto, etc.

Do we want to add this to the typehints?